src/main/java/seedu/address/logic/commands/PatientRemarkCommand.java

```diff
@@ -23,9 +24,9 @@ public class PatientRemarkCommand extends Command {
             + "by the index number used in the last patient listing. "
             + "Existing remark will be overwritten by the input.\n"
             + "Parameters: INDEX (must be a positive integer) "
-            + "r\\ [REMARK]\n"
+            + PREFIX_REMARK + "REMARK\n"
             + "Example: " + COMMAND_WORD + " 1 "
-            + "r/ Patient to follow up in 1 month.";
+            + PREFIX_REMARK + "Patient to follow up in 1 month.";
```

Comment:
Can change the example remark to something that follows our suggested usage of remarks in the UG, eg. family history of diabetes.